.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-  - "2.6"
```

Comment:
@EvaSDK Why did you remove 2.6 from `.travis.yml`? It's still in `tox.ini` and seems working fine at least for me.